docs/gettingstarted.rst

```diff
@@ -59,7 +60,7 @@ You can also easily deploy your own REANA cloud instance by using the
    $ reana-cluster init
    $ reana-cluster status
    $ # set environment variables for reana-client
-   $ eval $(reana-cluster env --incude-admin-token) # if you are admin
+   $ eval $(reana-cluster env --incude-admin-token) # since you are admin
```

Comment:
Not sure this one is necessary, but if we change the text, please also amend the other places where this phrase occurs:

```console
$ grep -Hnir 'if you are admin' reana* 
reana/docs/administratorguide.rst:197:   $ eval $(reana-cluster env --incude-admin-token) # if you are admin
reana/docs/gettingstarted.rst:62:   $ eval $(reana-cluster env --incude-admin-token) # if you are admin
```

We could also amend `./reana-cluster/docs/gettingstarted.rst` to make the text clearer there...